src/card.css

```diff
@@ -0,0 +1,14 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
```

Comment:
It's recommended to use 'align-items: center;' to center the content horizontally within the flex container.